### PR TITLE
Proofread Logs Bridge API

### PR DESCRIPTION
**Pull request by tigrannajaryan**
Contributes to https://github.com/open-telemetry/opentelemetry-specification/issues/3268

### CODE REVIEW

File: specification/logs/bridge-api.md [hunk 1]
```diff
@@ -33,42 +33,38 @@ data model.</b>
 
 The Logs Bridge API consist of these main classes:
 
-* LoggerProvider is the entry point of the API. It provides access to Loggers.
-* Logger is the class responsible for
-  emitting [Logs](./data-model.md#log-and-event-record-definition) as
-  LogRecords.
-
-LoggerProvider/Logger are analogous to TracerProvider/Tracer.
+* [LoggerProvider](#loggerprovider) is the entry point of the API. It provides access to Loggers.
```
Comment (MrAlias) [inline specification/logs/bridge-api.md:36]:
```suggestion
* [LoggerProvider](#loggerprovider) is the entry point of the API. It provides access to `Logger`s.
```

File: specification/logs/bridge-api.md [hunk 2]
```diff
@@ -88,34 +84,34 @@ returned as a fallback rather than returning null or throwing an exception, its
...
+which is not even observability-related). A `LoggerProvider` could also return a
+no-op `Logger` here if application owners configure the SDK to suppress telemetry
 produced by this library.
 - `version` (optional): Specifies the version of the instrumentation scope if
 the scope has a version (e.g. a library version). Example value: 1.0.0.
 - `schema_url` (optional): Specifies the Schema URL that should be recorded in
 the emitted telemetry.
 - `include_trace_context` (optional): Specifies whether the Trace Context should
-  automatically be passed on to the LogRecords emitted by the Logger. This
+  automatically be passed on to the `LogRecords` emitted by the `Logger`. This
```
Comment (MrAlias) [inline specification/logs/bridge-api.md:95]:
```suggestion
  automatically be passed on to the `LogRecord`s emitted by the `Logger`. This
```

**Review by MrAlias — APPROVED** [review]

**Review by reyang — APPROVED** [review]

File: specification/logs/bridge-api.md [hunk 3]
```diff
@@ -178,13 +173,13 @@ and is [one of the supported](README.md#direct-to-collector) log collection
 approaches.
 
 The Appender implementation will typically acquire a [Logger](#logger) from the
-global [LoggerProvider](#loggerprovider) at startup time, then
-call [Emit LogRecord](#emit-logrecord) for `LogRecords` received from the
-application.
+global [LoggerProvider](#loggerprovider) at startup time, then construct
+`LogRecord`s for received from the application and will call
```
Comment (djaglowski) [inline specification/logs/bridge-api.md:177]: Looks like this is missing a word at least. Maybe better as:
```suggestion
a `LogRecord` for each log received from the application, and then call
```

Comment (tigrannajaryan) [reply specification/logs/bridge-api.md]: Fixed.

**Review by djaglowski — APPROVED** [review]

**Review by jack-berg — APPROVED** [review]

**Pull request merged**